usaspending_api/awards/serializers.py

```diff
@@ -1,6 +1,7 @@
 from rest_framework import serializers
 from usaspending_api.awards.models import *
 from usaspending_api.accounts.serializers import AppropriationAccountBalancesSerializer
+from usaspending_api.references.serializers import *
```

Comment:
I personally like see the imported classes listed explicitly, just for readability. But it's just a style thing.